chef/cookbooks/monasca/recipes/common.rb

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright 2017 Fujitsu LIMITED
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitation.
+
+platform = node[:monasca][:platform]
+
+monasca_pkgs = platform[:packages]
+monasca_project = platform[:keystone][:tenant]
+monasca_roles = platform[:keystone][:roles]
+
+keystone_settings = KeystoneHelper.keystone_settings(node, @cookbook_name)
+
+register_auth_hash = {
+  user: keystone_settings["admin_user"],
+  password: keystone_settings["admin_password"],
+  tenant: keystone_settings["admin_tenant"]
+}
+
+log "monasca:common" do
+  message "monasca:common packagess are #{monasca_pkgs}"
+  level :debug
```

Comment:
Do we want `:debug` here? (similar in lines 36 and 40) 
Maybe it should be moved to separate variable in `default.rb` file.